pom.xml

```diff
@@ -208,6 +231,9 @@
                     <argLine>
                         -javaagent:"${settings.localRepository}/org/jmockit/jmockit/${jmockit.version}/jmockit-${jmockit.version}.jar"
                     </argLine>
+                    <excludes>
+                        <exclude>**/BuildWithEiffelStepRegressionTest.java</exclude>
+                    </excludes>
```

Comment:
Could we make the execution of these tests a bit more accessible? Ideally I think they should run via GH actions but not otherwise unless you ask for it, but I'd be fine with having it run by default but that it's easy to disable it when running locally.

Reply:
I average ~ 80 sec extra execution time for testing which brings my local average total time from ~ 6 min 10 sec to about 7 min 30 sec. So it is not a lot of extra time spent even thought this suite runs 41 tests. When running tests locally you could run `mvn clean test -Dtest=\!BuildWithEiffelStepRegressionTest` to exclude the regression suite.

Reply:
Oh, it's just a 40 s difference. In that case let's just run it all the time, and let power users can do the exclusion you describe if they're impatient. For quicker iterations one probably just runs individual tests from the IDE anyway.